source/tests/unit/ni_fake_non_ivi_service_tests.cpp

```diff
@@ -11,9 +11,12 @@ using namespace nifake_non_ivi_grpc;
 
 using ::testing::_;
 using ::testing::DoAll;
+using ::testing::HasSubstr;
```

Comment:
Consider: I think `using namespace ::testing;` would be a good alternative to all of these usings.